appcheck/firebase-appcheck/ktx/src/main/kotlin/com/google/firebase/appcheck/ktx/FirebaseAppCheck.kt

```diff
@@ -39,3 +44,12 @@ operator fun AppCheckToken.component1() = token
  * @return the expireTimeMillis of the [AppCheckToken]
  */
 operator fun AppCheckToken.component2() = expireTimeMillis
+
+internal const val LIBRARY_NAME: String = "fire-app-check-ktx"
+
+/** @suppress */
+@Keep
```

Comment:
nit: no need for this as keeps propagate from components
https://github.com/firebase/firebase-android-sdk/blob/eb4d215b5d8b7677cdc9e0d6dedf72025f4ad085/firebase-components/proguard.txt#L4-L5

Reply:
Removed